subworkflows/nf-core/bam_dedup_stats_samtools_umitools/tests/main.nf.test

```diff
@@ -0,0 +1,51 @@
+nextflow_workflow {
+
+    name "Test Workflow BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS"
+    script "../main.nf"
+    workflow "BAM_DEDUP_STATS_SAMTOOLS_UMITOOLS"
+    tag "subworkflows"
+    tag "subworkflows_nfcore"
+    tag "subworkflows/bam_dedup_stats_samtools_umitools"
+    tag "subworkflows/bam_stats_samtools"
+    tag "bam_dedup_stats_samtools_umitools"
+    tag "bam_stats_samtools"
+    tag "samtools"
+    tag "samtools/index"
+    tag "samtools/stats"
+    tag "samtools/idxstats"
+    tag "samtools/flagstat"
+    tag "umitools"
+    tag "umitools/dedup"
+
+    test("test_bam_dedup_stats_samtools_umitools") {
```

Comment:
the test name will anyway be displayed as per the name in L3.....recommend providing the test-dataset details here...so one can see what data is the test testing...thoughts ?